packages/amplify-graphql-model-transformer/.npmignore

```diff
@@ -0,0 +1,8 @@
+**/__mocks__/**
+**/__tests__/**
+src
+publish-notification-lambda
+rds-lambda
+rds-patching-lambda
```

Comment:
ignoring lambdas sources.
it appears that this package users are consuming `zip` files in `lib` not sources.